### DISCUSSION TITLE
[#1065550] /reload breaks UI actions

### DISCUSSION THREAD

**Discussion by TwistedAsylumMC** [Mojang/minecraft-editor#133] [Bugs]
Repro steps
1. Create a new project without any extensions
2. Use the /reload command after the world has loaded
3. Notice how all of the actions in the side bar and navigation menu are broken, as well as their keybimds

Observed result vs. what you actually expected to happen
As in vanilla (non-editor), /reload should reload all the scripts for any changes and continue to function as normal. Instead, reloading breaks all of the actions and you are stuck inside the project until you restart the client. This has also been reproduced by others in the [#editor-mode](https://discord.com/channels/523663022053392405/1084090299120373760) channel on the [BAO](https://discord.com/invite/wAtvNQN) discord.

Other helpful info
    Preview build version: 1.20.20
    Frequency: Always
    Local single-player, peer-to-peer, dedicated server (and how many people): Local single-player
    Device info): Windows 11 22621.1848

**Comment by minecraftchloe** [comment]
Oh no!!!! Alright, workaround for _now_ (we're working on the fix) is `Alt+Tab` or min/max your window to refresh the UI. It will be a couple of weeks until we can get the fix in. Our bad!!

**Comment by minecraftchloe** [comment]
Fixed with #137 !
